packages/hash/frontend/src/pages/[account-slug]/entities/[entity-uuid].page/entity-editor/links-section/link-table/cells/linked-with-cell.tsx

```diff
@@ -0,0 +1,109 @@
+import {
+  CustomCell,
+  CustomRenderer,
+  GridCellKind,
+} from "@glideapps/glide-data-grid";
+
+import {
+  getCellHorizontalPadding,
+  getYCenter,
+} from "../../../../../../../../components/grid/utils";
+import { drawTextWithIcon } from "../../../../../../../../components/grid/utils/draw-text-with-icon";
+import { drawCellFadeOutGradient } from "../../../../../../../../components/grid/utils/draw-cell-fade-out-gradient";
+import { LinkRow } from "../types";
+import { LinkedWithCellEditor } from "./linked-with-cell/linked-with-cell-editor";
+import { generateEntityLabel } from "../../../../../../../../lib/entities";
+import { InteractableManager } from "../../../../../../../../components/grid/utils/interactable-manager";
+
+export interface LinkedWithCellProps {
+  readonly kind: "linked-with-cell";
+  linkRow: LinkRow;
+}
+
+export type LinkedWithCell = CustomCell<LinkedWithCellProps>;
+
+export const renderLinkedWithCell: CustomRenderer<LinkedWithCell> = {
+  kind: GridCellKind.Custom,
+  needsHover: true,
+  isMatch: (cell: CustomCell): cell is LinkedWithCell =>
+    (cell.data as any).kind === "linked-with-cell",
+  draw: (args, cell) => {
+    const { rect, ctx, theme, spriteManager, hoverAmount, highlighted } = args;
+    const { linkAndTargetEntities, entitySubgraph, deleteLink } =
+      cell.data.linkRow;
+
+    ctx.fillStyle = theme.textHeader;
+    ctx.font = theme.baseFontStyle;
+
+    const yCenter = getYCenter(args);
+    const cellPadding = getCellHorizontalPadding();
+    const left = rect.x + cellPadding;
+
+    const linkAndTargetEntity = linkAndTargetEntities[0];
+
+    if (!linkAndTargetEntity) {
+      // draw empty value
+      ctx.fillStyle = "#91A5BA";
```

Comment:
Should this come from the theme?

Reply:
Yes, ideally, used `customColors` object for now, but we should eventually use the theme (to support switching themes)